client/source/index.jsx

```diff
@@ -7,6 +7,7 @@ import AppHolder from './AppHolder';
 
 
 // MAIN
+console.info('HOST:', `http://${process.env.SITE_URL}:${process.env.PORT}`);
```

Comment:
Just leaving some comments as I walk through what's happening, nothing to change!
This logs the path the app uses